src/runtime_src/core/tools/xbmgmt2/flash/xmc.cpp

```diff
@@ -537,7 +537,7 @@ bool XMC_Flasher::isXMCReady()
 
 bool XMC_Flasher::isBMCReady()
 {
-    bool bmcReady = (BMC_MODE() == 0x1);
+    bool bmcReady = (BMC_MODE() == 0x1) || (BMC_MODE() == 0x5);
```

Comment:
Could you please add a comment indicating what the values 0x1 and 0x5 represent.

Reply:
This code will change with the upcoming cleanup. This code should use enum but I wanted to keep the code change to the minimum for backporting